.github/workflows/0-on-demand.yaml

```diff
@@ -65,8 +65,6 @@ jobs:
   openwhisk:
     runs-on: ubuntu-22.04
     continue-on-error: false
-      strategy:
```

Comment:
I was trying to fix this in this PR: https://github.com/apache/openwhisk/pull/5420/files